.buildkite/run-cpu-test.sh

```diff
@@ -11,4 +11,23 @@ trap remove_docker_container EXIT
 remove_docker_container
 
 # Run the image and launch offline inference
-docker run --network host --env VLLM_CPU_KVCACHE_SPACE=1 --name cpu-test cpu-test python3 examples/offline_inference.py
+docker run -itd -v ~/.cache/huggingface:/root/.cache/huggingface --network host --env VLLM_CPU_KVCACHE_SPACE=4 --name cpu-test cpu-test
+
+# offline inference
+docker exec cpu-test bash -c "python3 examples/offline_inference.py"
+
+# async engine test
+#docker exec cpu-test bash -c "cd tests; pytest -v -s async_engine"
+
+# Run basic model test
+docker exec cpu-test bash -c "cd tests;
+  pip install pytest Pillow
+  rm -f __init__.py
```

Comment:
need this line otherwise this will trigger some missing module issue: `Import vllm._C error: No module named 'vllm._C'`

Reply:
https://stackoverflow.com/questions/41748464/pytest-cannot-import-module-while-python-can